src/httpclient.rs

```diff
@@ -23,16 +24,22 @@
 //!
 //! TODO
 
+pub mod auth;
+mod cache;
 pub mod data;
 pub mod proxy;
 
 use crate::data::project_id::ProjectId;
 use crate::data::renku_url::RenkuUrl;
 
 use self::data::*;
+use auth::Response;
+use auth::UserCode;
```

Comment:
suggestion: I just noticed this now, but should we group imports? like
```
use auth::{Response, UserCode};
```
this can also be nested.

Reply:
Sure, I don't have a strong opinion here. I use whatever rustfmt and lsp is doing. It would be nice to have some tool doing it, because it's easy to forget and do correctly when done manually? I now grouped them all in this file.